packages/devtools_app/lib/src/screens/memory/panes/allocation_tracing/allocation_profile_tracing_tree.dart

```diff
@@ -0,0 +1,274 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:vm_service/vm_service.dart';
+
+import '../../../../primitives/utils.dart';
+import '../../../../shared/common_widgets.dart';
+import '../../../../shared/table.dart';
+import '../../../../shared/table_data.dart';
+import '../../../../shared/theme.dart';
+import '../../../../shared/utils.dart';
+import '../../../profiler/cpu_profile_columns.dart';
+import '../../../profiler/cpu_profile_model.dart';
+import 'allocation_profile_tracing_view_controller.dart';
+
+const double _countColumnWidth = 130;
+
+/// Displays an allocation profile as a tree of stack frames, displaying
+/// inclusive and exclusive allocation counts.
+class AllocationTracingTree extends StatelessWidget {
+  const AllocationTracingTree({required this.controller});
+
+  final AllocationProfileTracingViewController controller;
+
+  @override
+  Widget build(BuildContext context) {
+    return ValueListenableBuilder<TracedClass?>(
+      valueListenable: controller.selectedTracedClass,
+      builder: (context, selection, _) {
+        Widget? errorColumn;
```

Comment:
instead of storing `errorColumn` in a var, just return from each if / else if statement since you are returning `errorColumn` anyway if non-null